[2025-02-28T19:36:01.481+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-02-28T19:34:00+00:00 [queued]>
[2025-02-28T19:36:01.488+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-02-28T19:34:00+00:00 [queued]>
[2025-02-28T19:36:01.489+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 1
[2025-02-28T19:36:01.500+0000] {taskinstance.py:1350} INFO - Executing <Task(SimpleHttpOperator): get_velib_data> on 2025-02-28 19:34:00+00:00
[2025-02-28T19:36:01.504+0000] {standard_task_runner.py:57} INFO - Started process 55865 to run task
[2025-02-28T19:36:01.508+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'api_to_db_dag', 'get_velib_data', 'scheduled__2025-02-28T19:34:00+00:00', '--job-id', '232', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmplyeknp2v']
[2025-02-28T19:36:01.510+0000] {standard_task_runner.py:85} INFO - Job 232: Subtask get_velib_data
[2025-02-28T19:36:01.560+0000] {task_command.py:410} INFO - Running <TaskInstance: api_to_db_dag.get_velib_data scheduled__2025-02-28T19:34:00+00:00 [running]> on host 2e469bce1432
[2025-02-28T19:36:01.627+0000] {taskinstance.py:1570} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='api_to_db_dag' AIRFLOW_CTX_TASK_ID='get_velib_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T19:34:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-28T19:34:00+00:00'
[2025-02-28T19:36:01.628+0000] {http.py:123} INFO - Calling HTTP method
[2025-02-28T19:36:01.638+0000] {base.py:73} INFO - Using connection ID 'velib_api_connection' for task execution.
[2025-02-28T19:36:01.819+0000] {taskinstance.py:1373} INFO - Marking task as SUCCESS. dag_id=api_to_db_dag, task_id=get_velib_data, execution_date=20250228T193400, start_date=20250228T193601, end_date=20250228T193601
[2025-02-28T19:36:01.851+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-02-28T19:36:01.893+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
